### PR TITLE
HyperLogLog support

### PR DESCRIPTION
**Pull request by lukebakken**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,17 +38,6 @@
...
-    {
-      "type": "package",
-      "package": {
-        "name": "basho/riak_pb",
-        "version": "2.1.3.0",
-        "dist": {
-          "url": "https://github.com/basho/riak_pb/archive/2.1.3.0.zip",
-          "type": "zip"
-        }
-      }
```
Comment (christophermancini) [inline composer.json]: Why remove it from composer and add it to git?

Comment (lukebakken) [reply composer.json]: It's how all the other clients pull in `riak_pb`. In addition, even a contributor to this project probably doesn't need / want to pull it in, either.

Comment (lukebakken) [reply composer.json]: PS check out #8, which I would like to get merged before this branch. I wanted to isolate the protobuf generation changes from the Hll changes.

File: composer.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     }
   ],
   "require": {
-    "basho/riak": "3.1.*",
+    "basho/riak": "dev-develop",
```
Comment (lukebakken) [inline composer.json:44]: @christophermancini I'm assuming this will stay on `dev-develop` until `riak-php-client` and this lib are released.

Comment (christophermancini) [reply composer.json:44]: Yes.

**Review by christophermancini — APPROVED** [review]
👍 :shipit:

**Pull request merged**